packages/venia-ui/lib/components/Checkbox/checkbox.css

```diff
@@ -20,7 +20,7 @@
     height: 1.25rem;
     justify-content: center;
     width: 1.25rem;
-    z-index: var(--base-z-index + 1, 1);
+    z-index: var(--base-z-index, 1);
```

Comment:
Good catch, this was certainly an error, and this does fix it.

However, I think while we're here, I'd prefer to set this to the lowest possible value, if at all. One of the following solutions would be preferable, I think:

- Set `z-index` directly to `1` here, **or**
- Render the icon element *after* the input element and delete this `z-index`

In the latter option, the DOM order alone will cause them to stack properly. Generally we want our DOM order to default to the most reasonable stacking, so that's probably the best choice here. But I would also accept the former solution.

Reply:
Yeah, I also agree removing the z-index is better.